features/url.yml

```
@@ -1,7 +1,8 @@
-draft_date: 2024-09-09
-name: URL Standard
-description: TODO
+name: URL
+description: The URL API allows developers to parse and interact with URL strings.
 spec: https://url.spec.whatwg.org/
```

We ought to link this to [the equivalent caniuse feature](https://caniuse.com/url), when it exists.

```suggestion
spec: https://url.spec.whatwg.org/
caniuse: url
```

Unfortunately, this also creates a bit of work for us because the generated status doesn't line up with caniuse. In this case it reveals that the Safari data for the properties (hash, host, etc.) are wrong in BCD. Would you open a collector PR on BCD for this? Most of this stuff seems to have landed in Safari 7.

---

@queengooborg can you follow up on this?

---

Under https://github.com/web-platform-dx/web-features/pull/1915 the BCD doesn't need to change now (though I'm curious if any of it's right), but we do need the caniuse key before merging this feature.

---

Ah, weird. It didn't make this outdated.